k8s/hyperconverged/tutorial-configure-openebs-gke.md

```diff
@@ -125,26 +159,69 @@ All you need to do, to use OpenEBS as persistent storage for your Stateful workl
 
 ```bash
-kubectl apply -f demo/jupyter/demo-jupyter-openebs.yaml
+root@ubuntu:~/demo_k8/openebs/k8s$ kubectl apply -f demo/jupyter/demo-jupyter-openebs.yaml
+deployment.apps "jupyter-server" created
+persistentvolumeclaim "jupyter-data-vol-claim" created
+service "jupyter-service" created
 ```
 
 The above command will create the following, which can be verified using the corresponding kubectl commands:
 
 - Launch a Jupyter Server, with the specified notebook file from github
-  (kubectl get deployments)
+  ```
+  root@ubuntu:~/demo_k8/openebs/k8s$  kubectl get deployments
+  NAME                                            DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
+  jupyter-server                                  1         1         1            1           2m
+  pvc-41964ed3-790a-11e8-a986-42010a80017b-ctrl   1         1         1            1           2m
+  pvc-41964ed3-790a-11e8-a986-42010a80017b-rep    3         3         3            3           2m
+  ```
 - Create an OpenEBS Volume and mounts to the Jupyter Server Pod (/mnt/data)
-  (kubectl get pvc)
-  (kubectl get pv)
-  (kubectl get pods)
-- Expose the Jupyter Server to external world via the http://NodeIP:32424 (NodeIP is any of the minion nodes external IP)
-  (kubectl get pods)
-
+  ```
+  root@ubuntu:~/demo_k8/openebs/k8s$ kubectl get pvc
+  NAME                     STATUS    VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS      AGE
+  jupyter-data-vol-claim   Bound     pvc-1827892f-7909-11e8-a986-42010a80017b   5G         RWO            openebs-jupyter   43s
+  ```
+  ```
+  root@ubuntu:~/demo_k8/openebs/k8s$ kubectl get pv
+  NAME                                       CAPACITY   ACCESS MODES   RECLAIM POLICY   STATUS    CLAIM                                   STORAGECLASS      REASON    AGE
+  pvc-1827892f-7909-11e8-a986-42010a80017b   5G         RWO            Delete           Bound     default/jupyter-data-vol-claim           openebs-jupyter     
+  ```
+  ```
+  root@ubuntu:~/demo_k8/openebs/k8s$ kubectl get pods
+  NAME                                                             READY     STATUS              RESTARTS   AGE
+  jupyter-server-68d4ddc48-mml47                                   1/1       Running		       	   0          2m
+  pvc-41964ed3-790a-11e8-a986-42010a80017b-ctrl-6877d997fb-msnxk   2/2       Running             0          2m
+  pvc-41964ed3-790a-11e8-a986-42010a80017b-rep-8f94c7b48-gdxbb     1/1       Running             0          2m
+  pvc-41964ed3-790a-11e8-a986-42010a80017b-rep-8f94c7b48-h5d9p     1/1       Running             0          2m
+  pvc-41964ed3-790a-11e8-a986-42010a80017b-rep-8f94c7b48-skssw     1/1       Running             0          2m
+  ```
+- Expose the Jupyter Server to external world via the http://NodeIP:32424 (NodeIP is any of the worker nodes external IP where application Pod is running)
```

Comment:
Display the Jupyter server by accessing the http... (Node IP is any worker nodes' external IP where the application pod is running).